content/installation/netcore/install/NetCoreInstallation.md

```diff
@@ -5,56 +5,38 @@ description: "Contrast .NET Core agent installation instructions"
 tags: "installation agent .NET Core windows linux"
 -->
 
-## The Basics
+## Install Overview
 
-To install the .NET agent, complete the following steps. The installation process for self-hosted applications also applies to IIS Express users.
+This document details the basic install of the .NET Core agent. The installation process for self-hosted applications also applies to IIS Express users.
 
+## Step 1 - Download Contrast Agent
+
+<details><summary><b>Download the Contrast .NET Core Agent From Contrast UI</b></summary>
```

Comment:
Redundant. This repeats the header. Should remove it.

Reply:
I changed the text to say "Download From Contrast UI".  
We want to keep this text as hideable/accordion to keep the page smaller and more readable.  In the near future, we will add another option here for Nuget.org downloads.